styles/main.css

```diff
@@ -0,0 +1,756 @@
+@import url('https://fonts.googleapis.com/css2?family=Oxygen:wght@300;400;700&display=swap');
+@font-face {
+    font-family: 'et-line';
+    src: url('/resources/fonts/et-line-font/et-line.eot');
+    src: url('/resources/fonts/et-line-font/et-line.eot?#iefix') format('embedded-opentype'), url('/resources/fonts/et-line-font/et-line.woff') format('woff'), url('/resources/fonts/et-line-font/et-line.ttf') format('truetype'), url('/resources/fonts/et-line-font/et-line.svg#et-line') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
```

Comment:
add font imports and definitions in a different file

Reply:
Done